### ISSUE TITLE
Anchor Box Optimization for Object Detection

### ISSUE THREAD

**Issue by NoahHalonen** [NoahHalonen/NoahHalonen-Capstone_Resources#19]
### Title
Anchor Box Optimization for Object Detection.

### URL
https://arxiv.org/abs/1812.00469

### Summary 
This paper introduces a method for optimizing anchor boxes in object detection frameworks. By treating anchor shapes as trainable variables, the proposed approach allows anchors to dynamically adapt to the data distribution and the network's learning capacity. This significantly simplifies the problem of designing anchor boxes, traditionally done through heuristic methods. The method can be implemented with stochastic gradient descent and integrated into any anchor box-based detection framework without affecting inference time or memory cost. Experiments on benchmark datasets such as Pascal VOC, MS COCO, and Brainwash demonstrate consistent improvements in detection accuracy.

### Key Points 
- Proposes a novel approach for optimizing anchor shapes during training, allowing for automatic adaptation to data and network characteristics.
- Implements optimization with minimal additional training cost and no impact on inference performance.
- Demonstrates consistent improvements in mean Average Precision (mAP) across various datasets and frameworks, including YOLOv2 and Faster-RCNN.
- Shows robustness to different anchor initialization methods and numbers of anchor shapes, greatly reducing the need for manual anchor design.
- Utilizes three training techniques for enhanced learning: soft assignment warm-up, online clustering warm-up, and batch normalization without shifting.

### Citation
@misc{zhong2020anchor,
      title={Anchor Box Optimization for Object Detection}, 
      author={Yuanyi Zhong and Jianfeng Wang and Jian Peng and Lei Zhang},
      year={2020},
      eprint={1812.00469},
      archivePrefix={arXiv},
      primaryClass={cs.CV}
}
